src/helpers.coffee

```diff
@@ -115,7 +115,7 @@ buildLocationData = (first, last) ->
     ]
 
 buildLocationHash = (loc) ->
-  "#{loc.first_line}x#{loc.first_column}-#{loc.last_line}x#{loc.last_column}"
+  "#{loc.range[0]}-#{loc.range[1]}"
```

Comment:
With the fixes to interpolation location data in `lexer.coffee`, a JSX comment test was breaking
 
Basically by using *exclusive* location data (eg `range`) for generating the location hashes used for tracking comments, the test passed since it distinguished two tokens' location data

And conceptually it makes sense to use exclusive location data here since it's a slightly stronger identifier of different location data (for tokens in this case)

Reply:
Can you add some version of this to a comment in the code? It would be good to remember this when working with comments in the future.